pc_ble_driver_py/ble_adapter.py

```diff
@@ -338,7 +346,10 @@ def service_discovery(self, conn_handle, uuid=None):
                 if response and "status" in response.keys():
                     if response["status"] == BLEGattStatusCode.success:
                         for char in response["characteristics"]:
-                            s.char_add(char)
+                            # Don't add repeat handles
+                            # See: https://github.com/NordicSemiconductor/pc-ble-driver-py/issues/219
```

Comment:
The issue you link to does not provide a lot of details. Either, the comments and code itself should be enough to explain what is going on, or the problem you are trying to solve is so complex that the resource you are linking to should provide all required details. Either remove the comment with the URL or add the same amount of detail as what you had in the PR.

Reply:
The main issue is I would like to address performing multiple `BLEAdapter.service_discovery` with multiple UUIDs in a program so I can discover single UUIDs on at a time. Currently, if you run `BLEAdapter.service_discovery` multiple times, you will most likely encounter this error message on the second `service_discovery`:
```
pc_ble_driver_py.exceptions.NordicSemiException: Failed to service_discovery. Error code: BLEGattStatusCode.invalid_handle
```
Then, if you retrieve this service list in your `BLEAdapter` object's `db_conns[conn_handle].service`, you will notice multiple instances of the same `uuid` with the same `start_handle` and `end_handle`. What my pull request was addressing was to mitigate these duplicates by checking if the UUIDs already existed within the `db_conns[conn_handle]` services, it's characteristics, and it's descriptors. This would allow multiple `BLEAdapter.service_discovery` methods to be run at any point, even with the same UUIDs so that multiple service handles do not exist and no errors will occur.